footer.php

```diff
@@ -28,6 +28,7 @@
 						array(
 							'theme_location' => 'footer',
 							'items_wrap'     => '%3$s',
+							'container'      => 'false',
```

Comment:
There was a container div here between the ul and li that was creating an error in [axe](https://www.deque.com/axe/) when testing,